frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -174,9 +170,7 @@ def add_store_credit_payments
         params.delete(:payment_source)
 
         # Return to the Payments page if additional payment is needed.
-        if @order.payments.valid.sum(:amount) < @order.total
-          redirect_to checkout_state_path(@order.state) and return
-        end
+        (redirect_to checkout_state_path(@order.state) and return) if @order.payments.valid.sum(:amount) < @order.total
```

Comment:
This looks terrible. Ignore rubocop and use previous version

Reply:
Ah, I understand. Okay I will undo.